### ISSUE TITLE
Tracks API allow for foreignAlbumId

### ISSUE THREAD

**Issue by tidusjar** [lidarr/Lidarr#732]
**Is your feature request related to a problem? Please describe.**
I have a need to display track information within Ombi without having the Artist in Lidarr. This allows the user to have a better understanding of the request being made.

**Describe the solution you'd like**
I'd like an API available similar to the [/track](https://github.com/lidarr/Lidarr/wiki/Track) API, but allow a `foreignAlbumId` as the artist might not be in Lidarr yet.

**Comment by ta264** [comment]
How would you want to deal with multiple releases?  There isn't a unique track list for a `foreignAlbumId`, there's one track list for each release.  By default, Lidarr will import a download against any of the track lists, so there's no guarantee the track list you show will be the track list you get.

It's possible to only allow a single release (i.e. request a particular track list) but this makes the download much more likely to fail to import because there's no good way to identify the particular release you're going to get from the indexer information before you've downloaded it.

**Comment by tidusjar** [comment]
Apologies regarding my lack of understanding around the music area. I was not aware of this. 

In that case the API could return all tracks for each release? Ombi would just display any if them as you stated, there is no guarantee, so I'd make the user aware of this.

**Comment by ta264** [comment]
It would be fairly easy to add this into the `v1/album/lookup` endpoint.  Would that work?

**Comment by tidusjar** [comment]
Yeah, I don't mind where the data comes from, I just need the data :)

**Comment by ta264** [comment]
Can you take a look at the build [here](https://ci.appveyor.com/project/Lidarr/lidarr/builds/23684307/job/pmwqrk5ta6cpucdr/artifacts) and see if that does what you need?  It only adds tracks to queries with an explicit musicbrainz id eg 
`/api/v1/album/lookup?term=lidarr%3A9fa7397e-7212-425f-8158-87c3bfb79602`

**Comment by tidusjar** [comment]
I'm currently running all my test services in docker, do you know how I can run those builds or just an example of the JSON response?

**Comment by ta264** [comment]
See [here](https://gist.githubusercontent.com/ta264/b480fe8768648afe3b47d1bd0c5d1e98/raw/ac1d1b6ae89ed60a0432110a7826250894e56eb8/gistfile1.txt)

Obviously subject to comment by @Qstick so don't take this as gospel just yet

**Comment by ta264** [comment]
We are going to make this a separate endpoint in the release after next (0.7; 0.6 is imminent)

**Comment by tidusjar** [comment]
Ok no problem. 
Thanks.

**Comment by tidusjar** [comment]
Is this still on the lidar/radar (see what I did there) for 0.7?

**Comment by Qstick** [comment]
@ta264 Is this good to close?

**Comment by bakerboy448** [comment]
@Qstick this landed in v1 I assume?
